userdocs/profiles.dev/docs/installer-docs/generate-local-manifests.md

```diff
@@ -30,3 +30,52 @@ pctl install \
   --out relative-path \
   <catalog name>/<profile>
 ```
+
+Consider the following installation:
+
+```
+pctl install --git-repository flux-system/flux-system nginx-catalog/nginx/v2.0.1
+generating profile installation from source: catalog entry nginx-catalog/nginx/v2.0.1
+installation completed successfully
+```
+
+Let's take a look inside:
+
+```
+tree nginx
+nginx
+├── artifacts
+│   └── bitnami-nginx
+│       ├── helm-chart
+│       │   ├── ConfigMap.yaml
+│       │   ├── HelmRelease.yaml
+│       │   └── HelmRepository.yaml
+│       ├── kustomization.yaml
+│       └── kustomize-flux.yaml
+└── profile-installation.yaml
+```
+
+This profile installs an `nginx` using a remote helm chart. The folders contain the following files in order:
```

Comment:
`an nginx` what?

Reply:
Uh... application? package? software? thing? :D

Reply:
I think either:

> This profile installs `nginx`

or

> This profile installs an `nginx` <insert descriptive thing>

"an nginx" is weird sounding

Reply:
Fair point. I'll think of something. :)